examples/i3c_target_set_ids.py

```diff
@@ -61,20 +59,32 @@ def main():
 
     PID_TO_SET = [0x02, 0x03, 0x04, 0x05, 0x06, 0x07]
     success, status = i3c_target.set_pid(PID_TO_SET)
-    print("PID assigned successfully" if success else "Could not assign the PID")
+    print(f"PID {PID_TO_SET} assigned successfully." if success else "Could not assign the PID")
 
     success, status = i3c_target.set_bcr(I3cTargetMaxDataSpeedLimit_t.MAX_DATA_SPEED_LIMIT, I3cTargetIbiCapable_t.NOT_IBI_CAPABLE, 
                                          I3cTargetIbiPayload_t.IBI_WITH_PAYLOAD, I3cTargetOfflineCap_t.OFFLINE_CAPABLE, 
                                          I3cTargetVirtSupport_t.VIRTUAL_TARGET_SUPPORT, I3cTargetDeviceRole_t.I3C_TARGET)
-    print("BCR assigned successfully" if success else "Could not assign the BCR")
+    
+    BCR_TO_SET = I3cBcrRegister_t() 
+    BCR_TO_SET.bits.maxDataSpeedLimitation = I3cTargetMaxDataSpeedLimit_t.MAX_DATA_SPEED_LIMIT.value
+    BCR_TO_SET.bits.ibiRequestCapable = I3cTargetIbiCapable_t.NOT_IBI_CAPABLE.value
+    BCR_TO_SET.bits.ibiPayload = I3cTargetIbiPayload_t.IBI_WITH_PAYLOAD.value
+    BCR_TO_SET.bits.offlineCapable = I3cTargetOfflineCap_t.OFFLINE_CAPABLE.value
+    BCR_TO_SET.bits.virtualTargetSupport = I3cTargetVirtSupport_t.VIRTUAL_TARGET_SUPPORT.value
+    BCR_TO_SET.bits.deviceRole = I3cTargetDeviceRole_t.I3C_TARGET.value
```

Comment:
Is this preferred over writing the BCR as just a normal array?

Reply:
We believe it is more readable for the user